packages/express/src/accountsExpress.js

```diff
@@ -22,6 +22,27 @@ const accountsExpress = (AccountsServer) => {
   ? res.status(500).jsonp(err.serialize().message)
   : res.status(500).jsonp({ message: err.message }));
 
+  router.use(async (req, res, next) => {
+    const accessToken = get(req.headers, 'accounts-access-token', undefined) || get(req.body, 'accessToken', undefined);
+    if (!isEmpty(accessToken)) {
+      try {
+        const user = await AccountsServer.resumeSession(accessToken);
+        // eslint-disable-next-line no-param-reassign
+        AccountsServer.userId = () => user.id;
```

Comment:
Because you don't always have the req object available but you can always import the singleton.